### PR TITLE
Updates to support 2020 for ACS5

### PR DESCRIPTION
**Pull request by sweatercomeback**
Working Tests!

### CODE REVIEW

**Comment by sweatercomeback** [comment]
@hancush This is based on and resolves the issues in PR#116.  I didn't want to jump in on someone else's fork.

File: census/core.py [hunk 1]
```diff
@@ -358,11 +358,11 @@ def zipcode(self, fields, zcta, **kwargs):
 
         return self.state_zipcode(fields, state_fips, zcta, **kwargs)
 
-    @supported_years(2019, 2018, 2017, 2016, 2015, 2014, 2013, 2012, 2011)
+    @supported_years(2020, 2019, 2018, 2017, 2016, 2015, 2014, 2013, 2012, 2011)
     def state_zipcode(self, fields, state_fips, zcta, **kwargs):
         return self.get(fields, geo={
             'for': 'zip code tabulation area:{}'.format(zcta),
-            'in': 'state:{}'.format(state_fips),
+            'regionin': 'state:{}'.format(state_fips),
```
Comment (hancush) [inline census/core.py:365]: Hey, @sweatercomeback, good to see you and thanks for this patch! Can you say more about this change?

Comment (sweatercomeback) [reply census/core.py:365]: This stackoverflow answer lead me to this solution after realizing commenting out the state_zipcode test allowed the tests to pass.   It looks like this was a change in the API moving ZCTA under state, thus requiring the "regionin" property to make it work. 
 
 https://stackoverflow.com/questions/50124143/error-unknown-unsupported-geography-heirarchy-when-querying-data-for-all-zcta5s

Comment (patwater) [reply census/core.py:365]: @hancush that work? Any further questions?

Comment (hancush) [reply census/core.py:365]: @sweatercomeback Thanks for the context! It doesn't seem like this change is consistent across available data years. With your change, 2020 works but all other years are raising an exception:

```
>>> c.acs5.state_zipcode(('NAME', 'B25034_010E'), states.MD.fips, '21218', year=2019)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/hannah/projects/cafwd-git/census/core.py", line 50, in wrapper
    return func(self, *args, **kwargs)
  File "/Users/hannah/projects/cafwd-git/census/core.py", line 363, in state_zipcode
    return self.get(fields, geo={
  File "/Users/hannah/projects/cafwd-git/census/core.py", line 313, in get
    return super(ACSClient, self).get(*args, **kwargs)
  File "/Users/hannah/projects/cafwd-git/census/core.py", line 159, in get
    merged_results = [merge(result) for result in zip(*all_results)]
  File "/Users/hannah/projects/cafwd-git/census/core.py", line 157, in <genexpr>
    all_results = (self.query(forty_nine_fields, geo, year, sort_by_geoid=sort_by_geoid, **kwargs)
  File "/Users/hannah/projects/cafwd-git/census/core.py", line 60, in wrapper
    result = func(self, *args, **kwargs)
  File "/Users/hannah/projects/cafwd-git/census/core.py", line 214, in query
    raise CensusException(resp.text)
census.core.CensusException: error: ambiguous geography "zip code tabulation area:21218". You must either specify a wildcard or full qualify it with state
```

Can you handle this and add a test that checks both 2019 and 2020 work as expected?

Comment (sweatercomeback) [reply census/core.py]: @hancush Updated!  I wasn't sure of the most elegant way to conditionally toggle between "regionin" and "in" in python, so I opted for the simplest version of it.

As for the test, I'm only testing that state_zipcode function.  Would you recommend testing all the functions with a hardcoded year?

Comment (hancush) [reply census/core.py]: @sweatercomeback Thanks so much! Simple is better than complex. 🙃 

Yes, I think extending the new test to check the other functions would be prudent. You should be able to reuse the original version of the ACS5 test cases, no? 

Can you also leave a comment in the function as to why the conditional is necessary?

Comment (sweatercomeback) [reply census/core.py]: Updated!

Comment (patwater) [reply census/core.py]: @hancush anything else needed? Thanks much!

**Review by hancush — APPROVED** [review]
Thanks so much, @sweatercomeback!

**Pull request merged**